eslint.config.js

```diff
@@ -135,12 +122,50 @@ export default config(
       "no-return-assign": ["warn", "always"],
       "no-script-url": "error",
       "no-sequences": ["error", { allowInParentheses: false }],
-      "no-throw-literal": "error",
+      "no-useless-return": "warn",
+      "no-undef-init": "warn",
       "no-unneeded-ternary": "warn",
       "no-useless-call": "error",
       "no-useless-computed-key": "warn",
       "no-useless-concat": "warn",
       "no-useless-rename": "error",
+      "no-var": "error",
+      "no-void": "error",
+      "object-shorthand": "warn",
+      "one-var": ["error", "never"],
+      "operator-assignment": "warn",
+      "prefer-arrow-callback": ["error", { allowNamedFunctions: true }],
+      "prefer-const": "error",
+      "prefer-exponentiation-operator": "error",
+      "prefer-named-capture-group": "warn",
+      "prefer-numeric-literals": "error",
+      "prefer-object-has-own": "error",
+      "prefer-object-spread": "error",
+      "prefer-regex-literals": ["error", { disallowRedundantWrapping: true }],
+      "prefer-rest-params": "error",
+      "prefer-spread": "error",
+      "prefer-template": "error",
+      radix: "error",
+      "require-unicode-regexp": "warn",
+      strict: "error",
+      "symbol-description": "warn",
+      "unicode-bom": ["error", "never"],
+      "vars-on-top": "warn",
+
+      // Code Quality "Params"
```

Comment:
**thought:** i am not usually a fan of these kinds of rules, but these are loose enough that i'm fine with this